ocw/lib/openqa.py

```diff
@@ -48,7 +40,7 @@ def __new__(cls, **kwargs):
     def __init__(self, **kwargs):
         kwargs.pop("server")
         self.__client = openqa_client.client.OpenQA_Client(server=self.server, **kwargs)
-        self.__client.session.verify = bool(os.environ.get('REQUESTS_CA_BUNDLE'))
+        self.__client.session.verify = False
```

Comment:
This is disabling TLS verification as well.

Reply:
> This is disabling TLS verification as well.

We can't verify every openQA instance with self-signed certs.

Reply:
OK, I see you are right. Thanks for the explanation!